apps/backend/app/api/routes/model_export.py

```
@@ -0,0 +1,261 @@
+        "algorithm": model.algorithm,
+        "problem_type": model.problem_type,
+        "feature_count": model.n_features,
+        "available_formats": formats,
+        "export_endpoints": {
+            "python": f"/api/v1/models/{model_id}/export/python",
+            "onnx": f"/api/v1/models/{model_id}/export/onnx",
+            "pmml": f"/api/v1/models/{model_id}/export/pmml",
+            "docker": f"/api/v1/models/{model_id}/export/docker"
+        },
+        "usage_examples": {
+            "curl_python": f"curl -H 'Authorization: Bearer $TOKEN' '{request.url.scheme}://{request.url.netloc}/api/v1/models/{model_id}/export/python' -o model_inference.py",
```

Ruff (0.14.4)

Undefined name `request`

(F821)